backend/libraries/jwt/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2021"
 
 [dependencies]
 candid = { workspace = true }
-jsonwebtoken = "9.2.0"
+ct-codecs = "1.1.1"
```

Comment:
We've been putting all the dependency versions in the workspace to avoid duplicates